docs/_templates/layout.html

```diff
@@ -0,0 +1,6 @@
+{% extends "!layout.html" %}
+
+  {% block menu %}
+    <a href="genindex.html">Keyword Index</a>
```

Comment:
I think it would be better to move this below `super()`.

Reply:
actually i placed it below at first, but then i saw the Toc got Index at the top ^___^
since the PR is merged and closed, it's waste to open a new PR just for one line switch. could you please help me change this?

Reply:
We have enough PR numbers, you can just open a new one.

Reply:
I put it at the top so because nobody would find it below the expanded toc, but the toc is collapsed on the left bar, so I think it is better to put it at the bottom there.